_posts/2020-12-05-data-quality-with-r.md

```diff
@@ -0,0 +1,2415 @@
+---
+layout: post
+title: An unexpected acquaintance? Data Validation in R
+author: filipwastberg
+excerpt: R is popular programming language and is famous for statistical analysis and stunning visualizations. But it is not limited to those tasks, here we take a look how it can be used to do data validation on a Snowflake database.
+date: 2020-12-17 06:30:00 +0100
+tags:
+ - R
+ - Data validation
+ - Snowflake
+ - ETL
+---
+
+## Do Data Scientists do Data Engineering?
+
+Data Engineering is a common task for any Data Scientist. Sure, we
+rarely do the “real” Data Engineering, e.g. loading data from a
+business system into a relational database. Rather, we work with data 
+that already is in a database, and sometimes (if we're lucky) it's put there by a Data Engineer. 
+We usually transform this data into an analytical format from which we can derive
+aggregations, visualizations and statistical models. In the start of any
+Data Science project these, jobs may be simple and not always of any significant
+concern but as your Data Science products get used more and more the
+importance of your data will increase.
+
+Therefore it might be a good idea for any Data Scientist to invest some time into data
+validation. There are many frameworks for data validation in different
+programming languages. But Data Scientists usually feel comfortable
+using tools they know, such as R and Python. R is a popular functional
+programming language for data analysis. However, even though data
+analysis is what R is famous for, the language can do a lot more besides 
+statistics. In this post we’ll take a look at the R
+package [`pointblank`](https://github.com/rich-iannone/pointblank) that have made data validation and data quality
+controls easy to do in R. `pointblank` also produces pretty tables for your
+data validation steps, and I love pretty things. But there's more to the package
+than being easy and pretty, it's a serious tool for any Data Scientist who wants
+to get control over their data.
+
+## Using Snowflake in R
+
+Data can come in many formats, but for this blog post I’ll focus on
+databases and especially the increasingly popular database `Snowflake`.
+
+Once you have created an odbc-DSN for your database it's usually straight 
```

Comment:
When writing a blog good idea to think about who is this for, who will read this. I recommend to think as wide as you can make sense, because people with already deep skills in the topic would probably not learn so much from it. In this case, reader might be a fellow technical person working in Solita or other companies, not necessarily deeply versed in R or RStudio for example.

This is rather big leap for reader to grasp what odbc-DSN is. Perhaps not necessary to open it up too much but perhaps include a link in this word for some good resource that explains it/how to set it up. For example, does it require a Windows machine? Something like simply this would suffice: https://db.rstudio.com/odbc/